internal/resources/fetching/fetchers/gcp/assets_fetcher_test.go

```diff
@@ -135,9 +135,10 @@ func (s *GcpAssetsFetcherTestSuite) TestFetcher_ElasticCommonData() {
 
 		asset := &GcpAsset{
 			Type:    fetching.CloudCompute,
-			SubType: inventory.ComputeInstanceAssetType,
+			SubType: "gcp-compute-instance",
```

Comment:
Maybe a good idea to add a constant for it?

Reply:
This is used only for test purposes. In code we use `getGcpSubType()` which outputs this value.